src/processHistorian.py

```diff
@@ -237,6 +265,7 @@ def work(self):
 
 if __name__ == "__main__":
     ph = ProcessHistorian()
+    hb_interval = ph.heartbeat_interval / 1000  # in seconds for time.sleep()
```

Comment:
If we are using a field from PH, maybe we should move all this code into a method of PH